Sources/Orbit/Components/Badge.swift

```diff
@@ -36,7 +36,6 @@ public struct Badge: View {
                     .padding(.vertical, Self.verticalPadding)
             }
             .padding(.horizontal, .xSmall)
-            .frameWidthAtLeastHeight()
```

Comment:
Haven't looked into how this is implemented, but do we really want to get rid of this one?

Reply:
It was mostly needed for the NotificationBadge to force circular shape for narrow numbers. This modifier has issue with multiline text, so it needed to be removed here (and probably removed completely later if the badges would merge again)